quiche/include/quiche.h

```diff
@@ -786,7 +786,7 @@ void quiche_h3_config_enable_extended_connect(quiche_h3_config *config, bool ena
 // Frees the HTTP/3 config object.
 void quiche_h3_config_free(quiche_h3_config *config);
 
-// A HTTP/3 connection.
+// An HTTP/3 connection.
```

Comment:
@ghedo why here use "An"? I think here must use "a", same with other comments.

Reply:
Because as far as I know that's how English works (see e.g. https://english.stackexchange.com/a/468483).